index.mdwn

```diff
@@ -1,15 +1,25 @@
-# What is this awesome window manager?
-
+<span class="align_center">
 **awesome** is a highly configurable, next generation framework window
 manager for X. It is very fast, extensible and licensed under the
 [GNU GPLv2 license](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
-
+</span>
+<span class="align_center">
 It is primarily targeted at power users, developers and any people dealing with
 every day computing tasks and who want to have fine-grained control on their
 graphical environment.
-
+</span>
```

Comment:
Block-level elements should not be put into an inline element (except when the inline element is `<a>`).
Markdown renders text blocks into `<p>` (a block-level element), `<span>` is an inline element. And changing that via `display: block` should only be done if absolutely necessary, as it breaks the semantics of those elements.

Use `<div>` if you want to wrap a `<p>`. And since you're applying the same style to both paragraphs, a single wrapper should be enough.